highlight/highlight
# feat(HIG-3693): update the error preview in session replay

## Summary


Adds a new right panel for error preview. 
<img width="317" alt="Screenshot 2023-01-31 at 1 50 56 PM" src="https://user-images.githubusercontent.com/17913919/215891634-c9bc886e-6053-4f8b-8e3c-5a1156ccb62a.png">


## How did you test this change?



1. Go to the session view, e.g. https://frontend-pr-4107.onrender.com/1/sessions/jYkGnE0bof8xvOnYHaTC3nVau0nq?page=1
2. Click on the error tab inside dev tools
3. Check that the right panel works ok.

## Are there any deployment considerations?


no
